Make.inc

```diff
@@ -486,7 +486,7 @@ OBJCOPY := $(CROSS_COMPILE)objcopy
 # file extensions
 ifeq ($(OS), WINNT)
   SHLIB_EXT := dll
-  STATICLIB_EXT := lib
+  STATICLIB_EXT := .a # MSVC is .lib
```

Comment:
if comments are on the same line in a makefile variable, the value will include the trailing space - this might not break anything, but keep it in mind